server/models/plugins/cardAttachment.js

```diff
@@ -0,0 +1,44 @@
+const mongoose = require("mongoose");
+const { ObjectId } = mongoose.Schema.Types;
+const basePlugin = require("./basePlugin");
+
+const cardAttachmentSchema = new mongoose.Schema({
+  files: [{ type: ObjectId, ref: "file", default: [] }],
+});
+
+cardAttachmentSchema.methods.get = async (pluginId) => {
+  const plugin = await this.findById(pluginId);
+  return plugin.files;
+};
```

Comment:
So, we're actually going to be given a data structure such as `{ body, params, query}) into the arguments. And verify plugin will confirm if the plugin exists. The get methods can just return the files in a response like 

`{ response: 200, data: files }`

We want to keep our responses generic to ensure we are able to have the plugin a bit more functional in terms of incoming data validation.

Reply:
Let's make that change where applicable in the rest of the code.